IronPythonTests/Setups.cs

```diff
@@ -0,0 +1,85 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Configuration;
+using System.Globalization;
+using System.IO;
+using System.Reflection;
+using NUnit.Framework;
+
+
+[SetUpFixture]
+public class Setup
+{
+    private string moduleRootFolder;
+    List<string> resolutionPaths;
+
+    [OneTimeSetUp]
+    public void RunBeforeAllTests()
+    {
+        var thisDir = Path.GetDirectoryName(Assembly.GetExecutingAssembly().Location);
+        var configPath = Path.Combine(thisDir, "TestServices.dll.config");
+
+        // Adjust the config file map because the config file
+        // might not always be in the same directory as the dll.
+        var map = new ExeConfigurationFileMap { ExeConfigFilename = configPath };
+        var config = ConfigurationManager.OpenMappedExeConfiguration(map, ConfigurationUserLevel.None);
+
+        var element = config.AppSettings.Settings["DynamoBasePath"];
+        moduleRootFolder = element?.Value ?? string.Empty;
+
+        if (string.IsNullOrEmpty(moduleRootFolder))
+        {
+            throw new Exception($"Missing DynamoBasePath in TestServices.dll.config. Please set the DynamoBasePath to a valid Dynamo bin folder. DynamoBasePath is set to {moduleRootFolder}");
+        }
+        else if (!File.Exists(Path.Combine(moduleRootFolder, "DynamoCore.dll")))
+        {
+            throw new Exception($"Invalid DynamoBasePath in TestServices.dll.config. Please set the DynamoBasePath to a valid Dynamo bin folder. DynamoBasePath is set to {moduleRootFolder}");
+        }
+
+        resolutionPaths = new List<string>
+        {
+            // Search for nodes
+            Path.Combine(moduleRootFolder, "nodes"),
+            Path.Combine(moduleRootFolder, "en-us")
```

Comment:
this allows resolving dynamo.resources.dll

Reply:
Dynamocore.dll is loaded from the test folder, the rest of dynamo runtime will be loaded from the test config path.
This might not work ok for other tests.